content/docs/design/visual_story@ko.md

```diff
@@ -0,0 +1,26 @@
+---
+$title: 눈이 즐거운 AMP 스토리 제작하기
+$order: 4
+components:
+    - video
+tutorial: true
+---
+
+{{ video('/static/video/tut_pets-story-complete.mp4', 374, 667, '/static/img/docs/tutorials/amp_story/story_poster.png', caption='완성된 AMP 스토리', align='right third', autoplay='true', loop='true' ) }}
+
+이 튜토리얼은 눈길을 끄는 이야기를 만들도록 도와주는 [amp-story](/docs/reference/components/amp-story.html) 컴포넌트를 소개한다. 튜토리얼을 끝내면 정보와 시각효과가 잘 어우러져서 독자들이 읽기편한 “애완동물이 주는 기쁨”이라는 이야기가 완성되어 있을 것이다. 
```

Comment:
Any links to docs should include the locale, so that it links to the locale's file.  You wouldn't want to be in a Korean page than taken to an English page.  So `/docs/reference/components/amp-story.html` should be `/ko/docs/reference/components/amp-story.html`